code/__HELPERS/unsorted.dm

```diff
@@ -257,7 +257,9 @@ Turf and target are seperate in case you want to teleport some distance from a t
 
 /obj/proc/atmosanalyzer_scan(datum/gas_mixture/air_contents, mob/user, obj/target = src)
 	var/obj/icon = target
-	user.visible_message("[user] has used the analyzer on [target].", "<span class='notice'>You use the analyzer on [target].</span>")
+	to_chat(user,"<span class='notice'>You use the analyzer on [target].</span>" )
+	if(!istype(user, /mob/dead/observer))
+		user.visible_message("<span class='notice'>[user] has used the analyzer on [target].</span>")
```

Comment:
Two things here:
- Consistency in verb tenses, I think we mostly use present tense
- I don't think ghosts necessarily need this message, there's no real need to pull it out of the visible message imho

```suggestion
	if(isliving(user))
		user.visible_message("<span class='notice'>[user] uses the analyzer on [target].</span>", "<span class='notice'>You use the analyzer on [target].</span>")
```

Reply:
The split was to fix the `You use the analyzer on [target]. message from appearing to anyone visible to the user.

Reply:
The second message will only be visible to the user, though.